### PR TITLE
[RHDM-739] optaweb-employee-rostering templates incomplete

### PR DESCRIPTION
**Pull request by errantepiphany**
[RHDM-739] optaweb-employee-rostering templates incomplete
https://issues.jboss.org/browse/RHDM-739

Signed-off-by: David Ward <dward@redhat.com>

Thanks for submitting your Pull Request!

Please make sure your PR meets the following requirements:

- [x] Pull Request title is properly formatted: `[RHDM-XYZ] Subject` or `[RHPAM-XYZ] Subject`
- [x] Pull Request contains link to the JIRA issue
- [x] Pull Request contains description of the issue
- [x] Pull Request does not include fixes for issues other than the main ticket
- [x] Attached commits represent units of work and are properly formatted
- [x] You have read and agreed to the Developer Certificate of Origin (DCO) (see `CONTRIBUTING.md`)
- [x] Every commit contains `Signed-off-by: Your Name <yourname@example.com>` - use `git commit -s`

### CODE REVIEW

**Comment by errantepiphany** [comment]
@spolti Can you please review, and hopefully approve? Thank you!

File: jboss-kie-optaweb-employeerostering/added/launch/jboss-kie-optaweb-employeerostering.sh [hunk 1]
```diff
@@ -16,11 +16,14 @@ function configure() {
...
+    # for persistence.xml property replacement
+    if [ "${OPTAWEB_EMPLOYEE_ROSTERING_PERSISTENCE_DATASOURCE}" != "" ]; then
+        JBOSS_KIE_ARGS="${JBOSS_KIE_ARGS} -Dorg.optaweb.employeerostering.persistence.datasource=${OPTAWEB_EMPLOYEE_ROSTERING_PERSISTENCE_DATASOURCE}"
+    fi
     if [ "${OPTAWEB_EMPLOYEE_ROSTERING_PERSISTENCE_DIALECT}" != "" ]; then
         JBOSS_KIE_ARGS="${JBOSS_KIE_ARGS} -Dorg.optaweb.employeerostering.persistence.dialect=${OPTAWEB_EMPLOYEE_ROSTERING_PERSISTENCE_DIALECT}"
     fi
-    if [ "${OPTAWEB_EMPLOYEE_ROSTERING_PERSISTENCE_DS}" != "" ]; then
-        JBOSS_KIE_ARGS="${JBOSS_KIE_ARGS} -Dorg.optaweb.employeerostering.persistence.datasource=${OPTAWEB_EMPLOYEE_ROSTERING_PERSISTENCE_DS}"
+    if [ "${OPTAWEB_EMPLOYEE_ROSTERING_PERSISTENCE_HBM2DDL_AUTO}" != "" ]; then
```
Comment (spolti) [inline jboss-kie-optaweb-employeerostering/added/launch/jboss-kie-optaweb-employeerostering.sh]: Persistence.ds is not needed?

**Comment by spolti** [comment]
just the comment above, besides that the PR looks ok.

File: jboss-kie-optaweb-employeerostering/added/launch/jboss-kie-optaweb-employeerostering.sh (hunk 1 above)
Comment (errantepiphany) [reply jboss-kie-optaweb-employeerostering/added/launch/jboss-kie-optaweb-employeerostering.sh]: The env var is _DATSOURCE is for employee rostering, to match the system property name that is being set. In the templates we do populate it with the _DS parameter, along with the _JNDI one.

Comment (spolti) [reply jboss-kie-optaweb-employeerostering/added/launch/jboss-kie-optaweb-employeerostering.sh]: got it.

**Pull request merged**